tests/virt_autotest/install_package.pm

```diff
@@ -76,7 +76,17 @@ sub install_package {
     #install qa_lib_virtauto
     if (check_var('ARCH', 's390x')) {
         lpar_cmd("zypper --non-interactive --gpg-auto-import-keys ref");
-        lpar_cmd("zypper --non-interactive in qa_lib_virtauto");
+        my $pkg_lib_data = "qa_lib_virtauto-data";
+        my $cmd          = "rpm -qa | grep $pkg_lib_data";
+        my $ret          = console('svirt')->run_cmd($cmd);
+        if ($ret == 0) {
+            lpar_cmd("zypper --non-interactive rm $pkg_lib_data");
+            lpar_cmd("zypper --non-interactive in qa_lib_virtauto");
```

Comment:
this same command shows up twice. it can have the correct result but if you move it out of 'if' loop, it makes the codes kind of simpler. others are LGTM.

Reply:
Got it, had changed code as your mentioned. Thank you so much for your great help.